app/controllers/Application.java

```diff
@@ -1135,18 +1136,23 @@ public static Result restFlowGraphData(String flowDefId) {
         flowPerfScore += jobPerfScore;
       }
 
+
       // Execution record
       JsonObject dataset = new JsonObject();
-      dataset.addProperty("flowtime", mrJobsList.get(mrJobsList.size() - 1).finishTime);
+      dataset.addProperty("flowtime", getFlowTime(mrJobsList.get(mrJobsList.size() - 1)));
```

Comment:
This is a change quite specific to Langoustine in fact. I am not changing anything for other schedulers as the getFlowTime method is keeping the old behaviour for jobs not scheduled with Langoustine. Imo this is part of the Langoustine integration, and thus under the scope of this PR, but it could indeed be moved to another PR, as it could work without this change.